core/src/main/java/io/confluent/rest/errorhandlers/NoJettyDefaultStackTraceErrorHandler.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright 2022 Confluent Inc.
- */
-
-package io.confluent.rest.errorhandlers;
-
-import java.io.IOException;
-import jakarta.servlet.RequestDispatcher;
-
-import org.eclipse.jetty.ee10.servlet.ServletContextRequest;
-import org.eclipse.jetty.http.HttpStatus;
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.servlet.http.HttpServletResponse;
-import org.eclipse.jetty.ee10.servlet.ErrorHandler;
-
-public class NoJettyDefaultStackTraceErrorHandler extends ErrorHandler {
-  public NoJettyDefaultStackTraceErrorHandler() {
-    super();
-    setShowServlet(false);
-  }
-
-  @Override
-  protected void generateAcceptableResponse(ServletContextRequest baseRequest,
-                                            HttpServletRequest request,
-                                            HttpServletResponse response,
-                                            int code, String message) throws IOException {
-    // set Exception to null to avoid exposing stack trace to clients
-    request.setAttribute(RequestDispatcher.ERROR_EXCEPTION, null);
```

Comment:
in Jetty 12 org.eclipse.jetty.server.handler.ErrorHandler , we don't need this because this is prevented by `setShowStacks` in `StackTraceErrorHandler.java`